tutorials/platform/android/javaclasswrapper_and_androidruntimeplugin.rst

```diff
@@ -0,0 +1,105 @@
+.. _doc_javaclasswrapper_and_androidruntimeplugin:
+
+Integrating with Android APIs
+=============================
+
+The Android platform has numerous APIs as well as a rich ecosystem of third-party libraries with wide and diverse functionality:
+
+- Push notifications
+- Analytics
+- Login 
+- Ads 
+- etc...
```

Comment:
Maybe this list should be just part of the previous paragraph, to keep it shorter? "..diverse functionality, like push notifications, analytics, authentication, ads etc."? Or similar. Don't care strongly about this however.